common/prettify.pm

```diff
@@ -1768,6 +1772,21 @@ sub BuildErrors ($)
      return @{$self->{OUTPUT}[0]->{BUILD_ERROR_COUNTER}};
 }
 
+sub get_host_from_log($)
+{
+  if ($host eq 'localhost') {
+    my $log = shift;
+    open(FILE, $log) or die "Can't open $log: $!";
+    my $bytes = 512;
```

Comment:
Why is this 512?

Reply:
For lack of a better way I know at this point to read a block at the beginning that contains <h3>Hostname</h3> and its value. (The 512-byte block is pretty safe to cover the Begin section and the Config section's <h3>Hostname</h3>, <h3>OS version (uname -a)</h3>, and part of <h3>Disk space information (df -k)</h3>.)

Reply:
Also, if the UTC time in the Begin section can be used as timestamp in the JUnit test report, it's very easy to parse from this block.

Reply:
Don't use an arbitrary value, just read from the filehandle until you're done and then close it.

Reply:
reverted back to Prettify::Config_HTML::host